calico-enterprise/_includes/components/ReqsSys.js

```diff
@@ -165,41 +165,6 @@ function NodeRequirementsEnt(props) {
   );
 }
 
-function KeyValueStore(props) {
```

Comment:
I'm not really sure how this works - is this function called from somewhere that we need to remove it as well?

Reply:
Yeah, it is failing to build. Need to track down where its called.